docs/tips.md

```diff
@@ -87,15 +87,14 @@ level, which applies to your GVCs mapped to that org.
 
 You can do this during the initial app setup, like this:
 
-1. Add the templates for `app` and `secrets` to `.controlplane/templates`
+1. Add the template for `app` to `.controlplane/templates`
 2. Ensure that the `app` template includes the `identity`
 3. Ensure that the `app` template is listed in `setup_app_templates` for the app in `.controlplane/controlplane.yml`
-4. Run `cpl apply-template secrets -a $APP_NAME` (one-time setup)
```

Comment:
The GitHub diff makes it hard to see, but the only thing that changed here was that line 4 was removed and every other line moved a number up.

Reply:
@coderabbitai resume

Reply:
@coderabbitai help

Reply:
@coderabbitai Please finish processing the PR.

Reply:
@coderabbitai See the notes in https://github.com/shakacode/control-plane-flow/pull/181#issuecomment-2128383281? They say it's still processing.